app/components/table-message/has-results.hbs

```diff
@@ -0,0 +1,10 @@
+{{!-- Partial Result Message --}}
+<TableMessage>
+    <p class="has-results">
+        Als u de persoon die u zoekt niet vindt, kijk na op spelfouten of probeer een andere zoekopdracht. 
+        Indien de persoon ontbreekt, 
+        <a href="mailto:{{config "contactEmail"}}" class="au-c-link">
+            + voeg een nieuwe persoon toe
+        </a>.
```

Comment:
Here, should we really be sending an email ? Don't we want to just set a link to the `person/new` page ..?